libconcorde_tsp_solver/CMakeLists.txt

```diff
@@ -31,6 +32,10 @@ install(DIRECTORY ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_INCLUDE_DESTINATION}/
     DESTINATION ${CATKIN_PACKAGE_INCLUDE_DESTINATION}
 )
 
-install(FILES ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}/concorde.a
+install(PROGRAMS ${CATKIN_DEVEL_PREFIX}/${CATKIN_PACKAGE_LIB_DESTINATION}/concorde.a
```

Comment:
This should probably stay `FILES`.

Reply:
Ok - I saw in another package that the .so libraries were also installed as program. Is this different for the static library .a? Then I switch to file. Please confirm.

Reply:
PROGRAMS sets the file permissions as executable, but libraries can't be executed so they shouldn't have +x.